lib/octokit/client/pull_requests.rb

```diff
@@ -276,6 +276,17 @@ def pull_request_files(repo, number, options = {})
       end
       alias pull_files pull_request_files
 
+      # Update a pull request branch
+      #
+      # @see https://developer.github.com/v3/pulls/#update-a-pull-request-branch
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @param number [Integer] Number of pull request
+      # @param options [Hash] Optional parameters (e.g. expected_head_sha)
+      # @return [Sawyer::Resource] Info regarding the update if successful
+      def update_pull_request_branch(repo, number, options = {})
+        put "#{Repository.path repo}/pulls/#{number}/update-branch", options
```

Comment:
What do you thinking of using `#boolean_from_response` here? Looking at the API, it appears that the response is not super useful - you just want to know if it worked or not.

```suggestion
        boolean_from_response(:put, "#{Repository.path repo}/pulls/#{number}/update-branch", options)
```